packages/plugin-data/package.json

```diff
@@ -28,7 +28,15 @@
                 "description": "fetch records using SOQL"
               },
               "tree": {
-                "description": "manipulate records using the tree API"
+                "description": "import/export records using the tree API",
+                "subtopics": {
```

Comment:
Why are these here? They aren't topics right?

Reply:
No they aren't topics.  It's a mistake.  I'll remove, thanks!